python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1692,8 +1692,15 @@ def run(self, ccdExposure, *, dnlLUT=None, bias=None, deferredChargeCalib=None,
         # (to make it simpler!)
         # Output units: electron (adu if doBootstrap=True)
         if self.config.doDeferredCharge:
-            self.log.info("Applying deferred charge/CTI correction.")
-            self.deferredChargeCorrection.run(ccdExposure, deferredChargeCalib)
+            if exposureMetadata["LSST ISR UNITS"] == "electron":
+                self.deferredChargeCorrection.config.useGains = False
+            else:
+                self.deferredChargeCorrection.config.useGains = True
```

Comment:
I was thinking this was superfluous, but no, we do need to be able to do the deferred charge application in adu (building calibrations, especially BFK) and electron (regular processing).

Reply:
Wait ... hold on ... this won't work in a real pipeline.  We cannot override configs in code, they are *frozen*.  (We probably should simulate this in the tests so this will appropriately fail, but we also want to be able to mess with configs in the tests so that's more of a refactor of the test code).  The `self.deferredChargeCorrection()` needs to be able to accept a runtime override of `useGains`.

Reply:
This is still broken and can’t work in a real pipetask